Description: Fix build problem in doc
 Let Makefile.am in doc not use DEPDIR
 Fix minor warnings in configure.ac
 .
 complexity (1.13-1) unstable; urgency=medium
 .
   *
Author: Boian Bonev <bbonev@ipacct.com>
Forwarded: not-needed
Last-Update: 2021-09-12

--- complexity-1.13.orig/doc/Makefile.am
+++ complexity-1.13/doc/Makefile.am
@@ -22,28 +22,10 @@ man_MANS            = $(exe).1
 optdefs             = $(top_srcdir)/src/opts.def
 BUILT_SOURCES       =
 
-if AMDEP
-TXI_DEP_FILE        = $(DEPDIR)/info-dep
-TXI_DEP             = -MF$(TXI_DEP_FILE) -MT$@
-$(TXI_DEP_FILE)     : example.texi
-
-MAN_DEP_FILE        = $(DEPDIR)/man-dep
-MAN_DEP             = -MF$(MAN_DEP_FILE)  -MT$@
-$(MAN_DEP_FILE)     : $(man_MANS)
-
-include $(DEPDIR)/info-dep
-include $(DEPDIR)/man-dep
-DEP_FILES           = $(DEPDIR)/info-dep $(DEPDIR)/man-dep
-$(DEP_FILES)        : $(DEPDIR)
-$(DEPDIR)           :
-	chmod 777 . && mkdir $@
-
-else
 TXI_DEP             =
 MAN_DEP             =
 DEP_FILES           =
 DEPDIR              =
-endif
 
 CLEANFILES          = *-stamp $(DEP_FILES) invoke-complexity.*
 TEXI2DVI_FLAGS      = --texinfo='@pagesizes 9.5in,7.0in'
--- complexity-1.13.orig/configure.ac
+++ complexity-1.13/configure.ac
@@ -11,7 +11,7 @@ dnl and will only look for install-sh in
 dnl AC_CONFIG_AUX_DIR([build-aux])
 dnl
 AC_CONFIG_MACRO_DIR([m4])
-AC_CONFIG_HEADER([config.h])
+AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_FILES([Makefile doc/Makefile lib/Makefile src/Makefile tests/Makefile])
 AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_SRCDIR([src/complexity.c])
@@ -19,12 +19,12 @@ AM_INIT_AUTOMAKE([gnu check-news 1.5 dis
 AC_USE_SYSTEM_EXTENSIONS
 gl_INIT
 
-AC_PROG_CC_C99
+AC_PROG_CC
 
 gl_EARLY
 ag_FIND_LIBOPTS
 [test -z "${ao_LIBS}" && \
-    ]AC_ERROR([libopts was not found])[
+    ]AC_MSG_ERROR([libopts was not found])[
 
 if autogen -MF/tmp/ag-bogus-$$ --help >/dev/null
 then ag_cv_ag_supports_mf=true
